examples/ios/QuickStartSwiftUI/QuickStart.swift

```diff
@@ -295,30 +290,62 @@ struct ItemDetailsView: View {
 
 /// Represents the login screen. We will just have a button to log in anonymously.
 struct LoginView: View {
-    @ObservedObject var auth: Auth
+    @EnvironmentObject var state: AppState
+    // Display an error if it occurs
+    @State var error: Error?
+
     var body: some View {
-        Button("Log in anonymously", action: auth.logIn).disabled(auth.isLoggingInOrOut)
+        VStack {
+            if let error = error {
+                Text("Error: \(error.localizedDescription)")
+            }
+            Button("Log in anonymously") {
+                state.shouldIndicateActivity = true
+                app.login(credentials: .anonymous).receive(on: DispatchQueue.main).sink(receiveCompletion: {
+                    switch ($0) {
+                    case .finished:
+                        state.shouldIndicateActivity = false
+                        break
+                    case .failure(let error):
+                        self.error = error
+                    }
+                }, receiveValue: {
+                    self.error = nil
+                    state.loginPublisher.send($0)
+                }).store(in: &state.cancellables)
+            }.disabled(state.shouldIndicateActivity)
+        }
     }
 }
 
+// MARK: ContentView
 /// The main screen that determines whether to present the LoginView or the ItemsView for the one group in the realm.
-struct ContentView: View {
-    @ObservedObject var auth: Auth
-    @ObservedObject var groupOwner: GroupOwner
-
-    init() {
-        let auth = Auth()
-        self.groupOwner = GroupOwner(auth: auth)
-        self.auth = auth
+@main
+struct ContentView: SwiftUI.App {
```

Comment:
This change is forcing me to upgrade

Reply:
Yes, I don't think we should display boilerplate app delegate code if we can avoid it.

Reply:
Me neither, I was just putting off the upgrade. Good to go now! 😁

Reply:
Update: haha at "good to go". This took two days but I finally got the iOS 14 simulator working.